doc/getting_started/index.rst

```diff
@@ -3,319 +3,409 @@
-applications.
+.. tabs::
+
+   .. group-tab:: Ubuntu
+
+      Use these commands to bring your Linux (Ubuntu) system up to date.
+      If you're using a different Linux distribution, use the appropriate
+      package manager for your OS.  (See :ref:`installation_linux` for
+      more information about other Linux distributions.)
 
-Depending on your operating system, you may or may not need to provide the
-``--user`` flag to the ``pip3`` command when installing new packages. This is
-documented throughout the instructions.
-See `Installing Packages`_ in the Python Packaging User Guide for more
-information about pip\ [#pip]_, including this `information on -\\-user`_.
+      .. code-block:: bash
 
-- On Linux, make sure ``~/.local/bin`` is on your :envvar:`PATH`
-  :ref:`environment variable <env_vars>`, or programs installed with ``--user``
-  won't be found\ [#linux_user]_.
+         sudo apt update
+         sudo apt upgrade
 
-- On macOS, `Homebrew disables -\\-user`_.
+   .. group-tab:: macOS
 
-- On Windows, see the Installing Packages information on ``--user`` if you
-  require using this option.
+      On macOS Mojave, you can manually check for updates by choosing
+      System Preferences from the Apple menu, then click Software Update (and
+      click Update Now if there are). For other macOS versions, see the
+      `Update macOS topic in Apple support
+      <https://support.apple.com/en-us/HT201541>`_.
 
-On all operating systems, the ``-U`` flag installs or updates the package if the
-package is already installed locally but a more recent version is available. It
-is good practice to use this flag if the latest version of a package is
-required.
+   .. group-tab:: Windows
+
+      On Windows, you can manually check for updates by selecting Start  > Settings  >
+      Update & Security  > Windows Update, and then select Check for updates.
+      If updates are available, install them.
 
 .. _install-required-tools:
 
-Install the required tools
-===========================
+.. rst-class:: numbered-step
+
+Install dependencies
+********************
+
+Next, use a package manager to install required support tools. Python 3
+and its package manager, pip, are used extensively by Zephyr for
+installing and running scripts used to compile, build, and run Zephyr
+applications.
+
+We'll also install Zephyr's multi-purpose west tool that simplifies
+getting the Zephyr project git repositories and external modules used by
+Zephyr.
+
+.. tabs::
+
+   .. group-tab:: Ubuntu
+
+      #. On Ubuntu, we can use the ``apt`` package manager to install
+         required support tools:
+
+         .. code-block:: bash
+
+            sudo apt install --no-install-recommends git cmake ninja-build gperf \
+              ccache dfu-util device-tree-compiler wget \
+              python3-pip python3-setuptools python3-tk python3-wheel xz-utils file \
+              make gcc gcc-multilib
```

Comment:
Are some of those perhaps for native_posix? Maybe they can be removed from GSG and into beyond-the-GSG if so?

Reply:
I don't think the extra packages are a problem (for now) since we're not quite sure if there are dependencies for STMicro or native_posix builds.  We can remove them later if/when we know.